### DISCUSSION TITLE
Proform Carbon TL

### DISCUSSION THREAD

**Discussion by davcho80** [cagnulein/qdomyos-zwift#2149] [Q&A]
Hello

I wonder if the Profrom Carbon TL would be compatible? I can't find it in the list, is it automatic not compatible or just not tested? The difference from the other TL is that it is a version without a screen, you must use your tablet which connects via Bluetooth to the treadmill.

https://www.canadiantire.ca/fr/pdp/tapis-roulant-pliable-proform-carbon-modele-2023-compatible-avec-ifit-1841654p.1841654.html?loc=plp

**Comment by cagnulein** [comment]
hi @davcho80 share me a nrfconnect screenshot from your phone and I will tell you for sure! nrfconnect is a free app on the store

**Comment by davcho80** [comment]
@cagnulein good I'm going to go buy the treadmill tomorrow and I'll send you the information as soon as it's assembled.

**Comment by cagnulein** [comment]
Sounds great, thank you!
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 21 feb 2024 alle ore 16:54 davcho80 ***@***.***>
ha scritto:

**Comment by davcho80** [comment]
Hello
![IMG_0005](https://github.com/cagnulein/qdomyos-zwift/assets/94725371/b9324964-9351-487e-bfcf-f5049ee9bb51)

I assembled the treadmill yesterday and this morning I took a screen print with nrfconnect. I already had qdomyoszwift installed for my Echelon bike. I opened the app and it automatically connected to the treadmill. He checked NordicTrack 10 in the settings. The speed and incline seems to indicate the correct information. I still have to test the inclination and automatic speed with the echelon app

**Reply by cagnulein** [reply]
OK, perfect, let me know

**Comment by ConorIA** [comment]
Glad you asked. I just assembled the same model but didn't take it for a run yet. Looking forward to using QZ with it!

**Reply by cagnulein** [reply]
@ConorIA let me know if you have any questions!

**Reply by ConorIA** [reply]
Just a quick report, as @davcho80 said, the app reads data no problem with Nordictrack 10 option set. 

I was running standalone (no Swift or anything). The controls on the tiles on QZ don't seem to change the settings on the Treadmill. I'm not sure if there is a need to pair the treadmill to the phone first somehow (e.g. via the iFit app?)

Separately, I was using the GitHub debug apk, but got kicked out after a "trial". I bought the app from the PlayStore. I assume the PlayStore version is fully unlocked?

**Reply by cagnulein** [reply]
> I was running standalone (no Swift or anything). The controls on the tiles on QZ don't seem to change the settings on the Treadmill. I'm not sure if there is a need to pair the treadmill to the phone first somehow (e.g. via the iFit app?)

nope you don't need the ifit app. for adding the control of speed and inclination in qz follow this with the ifit app and i will add it https://github.com/cagnulein/qdomyos-zwift/blob/master/docs/20_supported_devices_and_applications.md#android-device

> 
> 
> 
> Separately, I was using the GitHub debug apk, but got kicked out after a "trial". I bought the app from the PlayStore. I assume the PlayStore version is fully unlocked?

Yes the play store version is fully unlocked

**Reply by ConorIA** [reply]
Perfect. Followed the tutoial. Issue is here: https://github.com/cagnulein/qdomyos-zwift/issues/2208

**Comment by cagnulein** [comment]
Thanks I will check it ASAP

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 11 mar 2024 alle 01:00 Conor Anderson <
***@***.***> ha scritto:
